src/lib/bootloader/grub_install.rb

```diff
@@ -25,10 +25,10 @@ def execute(devices: nil, secure_boot: false)
 
       if devices
         devices.each do |dev|
-          Yast::Execute.locally(cmd + [dev])
+          Yast::Execute.on_target(cmd + [dev])
```

Comment:
This reminds me that I would start using `Yast::Execute` instead of the old `SCR::Execute` :)